src/main/java/com/coravy/hudson/plugins/github/GithubProjectProperty.java

```diff
@@ -58,10 +60,6 @@ public DescriptorImpl() {
             load();
         }
 
-        public boolean isApplicable(Class<? extends Job> jobType) {
```

Comment:
A downside is that the property will start appearing in other job types (External job monitor, Inheritance or whatever). BTW it should be safe enough since the property only produces an action